src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -0,0 +1,195 @@
+==== Housekeeping
+
+Notifications in the notification service have a global life span defined in the
+application.properties located in * `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/application.properties`
+Passed that time, they will be removed from the database.
+
+Can be defined:
+
+- The call frequency of the housekeeping defined by a CRON expression:
+----
+# This cron expression determines the housekeeping call frequency.
+# Default value is 10 minutes: this will invoke the housekeeping mechanism
+# to remove every notifications whose scheduled removal time is up.
+notifications.housekeeping.cronexpression=*/10 * * * *
+----
+
+- The life span of the notifications in seconds:
+----
+# Automatic remove notification delay (in seconds).
+# The time at which notifications will be stored in the database
+# Set this time to 0 if you don't want the notifications to be removed.
+# Default value is 604800 seconds = 1 week
+notifications.housekeeping.removedelay=604800
+----
+
+[[_notification_methods]]
+==== Notification methods
+
+[[_email_notification_method]]
+===== Email
+
+The Email notification method uses the Simple Mail Transfer Protocol to send emails.
+The notification service requires a valid SMTP configuration in order to send emails.
+These settings are specified in the properties file located at:
+ `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/email.properties`
+
+By default Transport Layer Security is active, change to false to deactivate it:
+----
+mail.smtp.starttls.enable=true
+----
+Authentication to the SMTP server is enabled by default.
+----
+mail.smtp.auth=true
+----
+
+Provide the server port used to connect
+----
+mail.smtp.port=
+----
+
+The credentials used to connect
+----
+mail.smtp.username=
+mail.smtp.password=
+----
+
+The host of the SMTP server
+----
+mail.smtp.host=
+----
+
+The email used to broadcast the emails is defined in the application.properties file,
```

Comment:
I don't understand this explanation "go to Housekeeping". This sentence is not english., and even written in french "Allez au housekeeping", this would not be an acceptable sentence for a documentation in french.  You could say go to the Housekeeping **section** for further explanation. But even then I don't understand one bit what you want to say. In what sense the housekeeping section is relevant here ???

A single email address is used by the notification service to broadcast emails.

Reply:
It was to link the location of the .property file that is specified a couple of lines above.
As it brings confusion I used your first sentence and wrote again the path to the file.